backend/climateconnect_main/settings.py

```diff
@@ -34,6 +34,7 @@
 
 CUSTOM_APPS = [
     'climateconnect_api',
+    'organization'
```

Comment:
Why do you register organization as a custom app?

Reply:
Kept organization as a separate app directory in django for better code management. When you do that you have to add a directory to CUSTOM_APPS to allow other directories to recognize database models, functions and APIs. Read more: https://docs.djangoproject.com/en/3.0/intro/tutorial01/#creating-the-polls-app